docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -57,3 +57,144 @@ If you want to self-host the Adyen app, reach out to [our team <Mail size={12}/>
     - `AUTHORIZATION_ACTION_REQUIRED`
     - `CHARGE_FAILURE`
     - `AUTHORIZATION_FAILURE`
+- [Webhook settings > Delayed Capture](#duplicated-chargedamount-in-saleor) is disabled in Adyen Dashboard
+- Webhook in Adyen was created with all options (additional settings) described in [Saleor docs](./configuration.mdx#webhook-configuration)
+- App supports following notification events from Adyen, other types of events will be ignored:
```

Comment:
Add this in a new sub-cathegory under 'assumptions' and move the "other types of events will be ignored:" part under the event list, in a caution! box